src/plugins/core/conditional_format.ts

```diff
@@ -276,6 +276,8 @@ export class ConditionalFormatPlugin
             "LessThan",
             "LessThanOrEqual",
             "NotContains",
+            "Equal",
+            "NotEqual",
```

Comment:
That means that commands that were previously valid will now be dropped. To be 100% complete, we should have a migration. 

Well the worth case is that the user is now missing some CF, if he had some CF `isEqual` with no value. That's very niche, and won't break any sheet. So we probably can skip the migrations. But I would still target master instead of 15.0.

Reply:
I've retargeted the PR to master 🙂